Makefile

```diff
@@ -97,3 +97,19 @@ endif
 
 $(FNAME_$(TARGET)): $(SOURCES) $(XFILES)
 	$(CXX) $^ -std=c++98 $(CFLAGS_G) $(MOREFLAGS) $(OPTFLAGS) $(CPPFLAGS) $(CFLAGS) $(CXXFLAGS) $(LFLAGS) -o$@
+
+ifneq ($(findstring linux,$(targetmachine)),)
+  ifeq ($(TARGET),gtk)
+install:
+	mkdir -p $(DESTDIR)$(prefix)/bin
+	mkdir -p $(DESTDIR)$(prefix)/share/applications
```

Comment:
If someone just types 'make install' with no other arguments, they'll end up with a /share in the root directory. Is that the expected behavior?

Reply:
Maybe we could default to /usr if it's not defined?

Reply:
Sounds good to me. This requires no more than `prefix ?= /usr` somewhere above this.